url835a90ed59_aschi_regularLanguages_tgz-pJ8-ch_zhaw_regularLanguages_evolution_runners_EvolvingLocalDivisable5LanguageJ8/src/ch/zhaw/regularLanguages/evolution/runners/EvolvingGlobalDivisable5LanguagePSScaling.java

```diff
@@ -0,0 +1,126 @@
+package ch.zhaw.regularLanguages.evolution.runners;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.List;
+
+import ch.zhaw.regularLanguages.evolution.initialisation.EvolvingGlobalProblemSetInitialisation;
+import ch.zhaw.regularLanguages.helpers.Logger;
+
+public class EvolvingGlobalDivisable5LanguagePSScaling {
+
+	public static void main(String[] args) {
+		EvolvingGlobalProblemSetInitialisation starter = new EvolvingGlobalProblemSetInitialisation();
+		starter.initLanguage(new char[] { '0', '1' }, 10, "(0|101|11(01)*(1|00)1|(100|11(01)*(1|00)0)(1|0(01)*(1|00)0)*0(01)*(1|00)1)*");
+
+		int solutionFoundCounter = 0;
+		int noSolutionFound = 0;
+		List<Long> cycleCount = new LinkedList<Long>();
+		long tmpCycle;
+		long timeStamp;
+
+		int[] problemCount = new int[25];
+		int[] candidatesCount = new int[1];
+		int[] noCycles = new int[2];
+
+		problemCount[0] = 3;
+		problemCount[1] = 6;
+		problemCount[2] = 9;
+		problemCount[3] = 12;
+		problemCount[4] = 15;
+		problemCount[5] = 18;
+		problemCount[6] = 21;
+		problemCount[7] = 24;
+		problemCount[8] = 27;
+		problemCount[9] = 30;
+		problemCount[10] = 33;
+		problemCount[11] = 36;
+		problemCount[12] = 39;
+		problemCount[13] = 42;
+		problemCount[14] = 45;
+		problemCount[15] = 48;
+		problemCount[16] = 51;
+		problemCount[17] = 54;
+		problemCount[18] = 57;
+		problemCount[19] = 60;
+		problemCount[20] = 63;
+		problemCount[21] = 66;
+		problemCount[22] = 69;
+		problemCount[23] = 72;
+		problemCount[24] = 75;
+
+		candidatesCount[0] = 50;
+
+		noCycles[0] = 250;
+		noCycles[1] = 500;
+
+		int pc = 0;
+		int cc = 0;
+		int nc = 0;
+
+		for (int x = 0; x < 10; x++) {
+			System.out.println("x:" + x);
+			for (int n = 0; n < 25; n++) {
+				DateFormat df = new SimpleDateFormat("yyyy-MM-dd_HH_mm_ss");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.